cluster-autoscaler/cloudprovider/hetzner/hetzner_cloud_provider.go

```diff
@@ -226,6 +251,51 @@ func BuildHetzner(_ config.AutoscalingOptions, do cloudprovider.NodeGroupDiscove
 			targetSize:         len(servers),
 			clusterUpdateMutex: &clusterUpdateLock,
 		}
+
+		// If a placement group was specified, check with the API to see if it exists
+		if manager.clusterConfig.IsUsingNewFormat && placementGroupRef := manager.clusterConfig.NodeConfigs[spec.name].PlacementGroup; placementGroupRef != nil {
```

Comment:
Short if statement can not be combined with logical operator && in this way. Anyway, it would be cleaner to move it inside the if-block.
```suggestion
		if manager.clusterConfig.IsUsingNewFormat {
			placementGroupRef := manager.clusterConfig.NodeConfigs[spec.name].PlacementGroup
			if placementGroupRef == "" {
				continue
			}
```